### PR TITLE
Bump subversion plugin dep

### PR DESCRIPTION
**Pull request by timja**
Fixes PCT on Java 11
https://github.com/jenkinsci/bom/pull/401

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -64,18 +64,17 @@
...
+        <subversion-plugin.version>2.14.0</subversion-plugin.version>
     </properties>
     <dependencyManagement>
         <dependencies>
             <dependency>
                 <groupId>io.jenkins.tools.bom</groupId>
-                <artifactId>bom-2.176.x</artifactId>
-                <version>7</version>
+                <artifactId>bom-2.222.x</artifactId>
+                <version>21</version>
```
Comment (timja) [inline pom.xml:77]:
```suggestion
                <version>22</version>
```

**Review by basil — APPROVED** [review]

**Comment by timja** [comment]
@bitwiseman @dwnusbaum @car-roll

File: pom.xml [hunk 2]
```diff
@@ -67,15 +67,14 @@
         <jenkins.version>2.222.4</jenkins.version>
         <java.level>8</java.level>
         <useBeta>true</useBeta>
-        <subversion-plugin.version>2.13.0</subversion-plugin.version>
-        <workflow-multibranch-plugin.version>2.21</workflow-multibranch-plugin.version>
+        <subversion-plugin.version>2.14.0</subversion-plugin.version>
```
Comment (dwnusbaum) [inline pom.xml:70]: Maybe `subversion` should be added to the BOM.

**Review by dwnusbaum — APPROVED** [review]

**Review by car-roll — APPROVED** [review]

**Comment by timja** [comment]
🚢 @car-roll  @dwnusbaum ?

**Pull request merged**